.gitignore

```diff
@@ -1,4 +1,5 @@
-out/
```

Comment:
Why the change? Does the old gitignore not work? We should figure out what should be in the gitignore.